classes/class-aal-api.php

```diff
@@ -107,19 +106,18 @@ public function insert( $args ) {
 			$args['user_caps'] = 'administrator';
 		
 		// Make sure for non duplicate.
-		$check_duplicate = $wpdb->get_row(
+		$check_duplicate = $wpdb->get_row( // !!!
 			$wpdb->prepare(
-				'SELECT `histid` FROM %1$s
-					WHERE `user_caps` = \'%2$s\'
-						AND `action` = \'%3$s\'
-						AND `object_type` = \'%4$s\'
-						AND `object_subtype` = \'%5$s\'
-						AND `object_name` = \'%6$s\'
-						AND `user_id` = \'%7$s\'
-						AND `hist_ip` = \'%8$s\'
-						AND `hist_time` = \'%9$s\'
+				'SELECT `histid` FROM `' . $wpdb->activity_log . '`
+					WHERE `user_caps` = \'%s\'
+						AND `action` = \'%s\'
+						AND `object_type` = \'%s\'
+						AND `object_subtype` = \'%s\'
+						AND `object_name` = \'%s\'
+						AND `user_id` = \'%s\'
+						AND `hist_ip` = \'%s\'
+						AND `hist_time` = \'%s\'
```

Comment:
The escaped quotes can be removed here as `prepare()` will handle them. so any `\'%s\'` should just be used as: `%s`